zerver/webhooks/dropbox/doc.md

```diff
@@ -6,9 +6,10 @@
 
 Next, log into your account on [Dropbox](https://www.dropbox.com), go
 [there](https://www.dropbox.com/developers/apps) and click **Create app**.
-Follow the steps, and in **Webhooks** section add the webhook address you created above.
+Follow the steps, and in **Webhooks** section, click **Generate** button  below **Generated access token** section.
```

Comment:
in *the* Webhooks section, click *the* Generate button, below *the* Generated access token section.

Reply:
Redundant space between button and below.